purchase_order_line_schedule/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,9 @@
+This module allows users to introduce a delivery schedule associated to
+a purchase order line. A schedule is used when a quantity on an order line
+is requested to be delivered in multiple dates.
+
+The Delivery schedule allows you to specify the number of dates in
+which the total quantity of the order line will be delivered from the vendor.
+
+By default, the system copies the total quantity and scheduled date
+of the original purchase line into the first delivery schedule line.
```

Comment:
Minor changes in text:

This module allows users to associate a delivery schedule to
a purchase order line. The schedule is used when quantities on an order line
are requested to be delivered on multiple dates.

The delivery schedule allows you to specify the dates on
which the quantities of the order line will be delivered from the vendor.

By default, the system copies the total quantity and scheduled date
of the original purchase line into the first delivery schedule line.

Reply:
Fixed.